.github/ISSUE_TEMPLATE/linkcheck_report.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 🐛 Weekly linkcheck failed
+about: Use this template for checking broken links
```

Comment:
Will this issue template show up as an option when someone tries to open an issue on github? I think we should avoid that.